packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -387,16 +387,23 @@ export class GCFBootstrapper {
     installationId: number,
     wrapOptions?: WrapOptions
   ): Promise<Octokit> {
-    // See: https://github.com/probot/probot/issues/1003
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
     const cfg = await this.getProbotConfig(wrapOptions?.logging);
-    return new cfg.Octokit!({
-      auth: {
-        appId: cfg.appId,
-        privateKey: cfg.privateKey,
-        installationId,
-      },
-    });
+    const opts = {
+      appId: cfg.appId,
+      privateKey: cfg.privateKey,
+      installationId,
+    };
+    if (wrapOptions?.logging) {
+      const LoggingOctokit = Octokit.plugin(LoggingOctokitPlugin)
```

Comment:
The Probot Options types seemed to conflict with our approach of instantiating Octokit ourselves.

Reply:
we will want to test our cron based applications.

Reply:
I tested it and failed🤣🤣